### PR TITLE
DEVTOOLS-683: Add authToken string to OAuthToken class

### PR DESCRIPTION
**Pull request by jord98716**
It would be useful to be able to get the oAuthToken string from the OAuthToken class, especially when a service needs to use an oauth token string to connect to other services.

### CODE REVIEW

**Comment by jord98716** [comment]
cc: @beckje01 @jeff-blaisdell

**Comment by jeff-blaisdell** [comment]
With the exception of the coverage dropping the change looks good to me.  However, I differ to someone closer to the project as to whether there is a good reason to not have the token stored.

File: ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/BearerTokenAuthHandler.java [hunk 1]
```diff
@@ -24,32 +24,36 @@ public BearerTokenAuthHandler(TokenValidator validator) {
 	@Override
 	public void handle(Context ctx) throws Exception {
 		String authHeader = ctx.getRequest().getHeaders().get(HttpHeaderNames.AUTHORIZATION);
-		if (authHeader != null && authHeader.startsWith("Bearer")) {
-			//Confirm token is valid
+		String token = ctx.getRequest().getQueryParams().get("authToken");
```
Comment (beckje01) [inline ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/BearerTokenAuthHandler.java]: Whats this bit for?

Comment (jord98716) [inline ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/BearerTokenAuthHandler.java]: In the platform api service that is under development, the url for getting a clientconn websocket is http://hostname/ws?authToken=XXXXX (a GET request). I'm using ratpack-bearer-auth to authenticate on that url. If necessary, I could change it to a POST and use the conventional Bearer auth header, but I was hoping to not have to do that.

Comment (beckje01) [inline ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/BearerTokenAuthHandler.java]: You can pull the token off with a different handler that is very not standard and I don't want to put that in the main lib.

Comment (jord98716) [inline ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/BearerTokenAuthHandler.java]: No problem, I'll change that.

**Comment by beckje01** [comment]
👍

**Pull request merged**